sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1,5871 +1,6389 @@
+                    "tooltip": "",
+                    "hideLabel": false,
+                    "tabindex": "",
+                    "disabled": false,
+                    "autofocus": false,
+                    "widget": {
+                        "type": "input"
+                    },
+                    "validateOn": "change",
+                    "validate": {
+                        "required": false,
+                        "custom": "",
+                        "customPrivate": false,
+                        "strictDateValidation": false,
+                        "multiple": false,
+                        "unique": false
+                    },
+                    "conditional": {
+                        "show": null,
+                        "when": null,
+                        "eq": ""
+                    },
+                    "allowCalculateOverride": false,
+                    "showCharCount": false,
+                    "showWordCount": false,
+                    "allowMultipleMasks": false,
+                    "addons": [],
+                    "inputType": "hidden",
+                    "id": "el0vre7"
+                },
+                {
+                    "label": "Field of study code",
+                    "labelPosition": "top",
+                    "labelWidth": "",
+                    "labelMargin": "",
+                    "placeholder": "",
+                    "description": "",
+                    "tooltip": "",
+                    "prefix": "",
+                    "suffix": "",
+                    "widget": {
+                        "type": "input"
+                    },
+                    "inputMask": "",
+                    "displayMask": "",
+                    "allowMultipleMasks": false,
+                    "customClass": "",
+                    "tabindex": "",
+                    "autocomplete": "",
+                    "hidden": false,
+                    "hideLabel": false,
+                    "showWordCount": false,
+                    "showCharCount": false,
+                    "mask": false,
+                    "autofocus": false,
+                    "spellcheck": true,
+                    "disabled": true,
+                    "tableView": true,
+                    "modalEdit": false,
+                    "multiple": false,
+                    "persistent": true,
+                    "inputFormat": "plain",
+                    "protected": false,
+                    "dbIndex": false,
+                    "case": "",
+                    "truncateMultipleSpaces": false,
+                    "encrypted": false,
+                    "redrawOn": "",
+                    "clearOnHide": true,
+                    "customDefaultValue": "",
+                    "calculateValue": "value = data.fieldOfStudyCodes.filter((d) => d.sabcCode === data.sabcCodeSelected && d.levelOfStudyCode === data.levelOfStudyCode)[0][\"federalStudyCode\"];",
```

Comment:
Same here. array.find()

Reply:
Updated.

Reply:
The input can be changed in the same way as the others and the find will be not needed.

Reply:
Updated.